includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -144,13 +144,70 @@ function perflab_aao_get_autoloaded_options_table() {
 	$autoload_summary = perflab_aao_query_autoloaded_options();
 
 	$html_table = sprintf(
-		'<table class="widefat striped"><thead><tr><th scope="col">%s</th><th scope="col">%s</th></tr></thead><tbody>',
-		__( 'Option Name', 'performance-lab' ),
-		__( 'Size', 'performance-lab' )
+		'<table class="widefat striped"><thead><tr><th scope="col">%s</th><th scope="col">%s</th><th scope="col">%s</th></tr></thead><tbody>',
+		esc_html__( 'Option Name', 'performance-lab' ),
+		esc_html__( 'Size', 'performance-lab' ),
+		esc_html__( 'Action', 'performance-lab' )
 	);
 
+	$nonce = wp_create_nonce( 'perflab_aao_update_autoload' );
 	foreach ( $autoload_summary as $value ) {
-		$html_table .= sprintf( '<tr><td>%s</td><td>%s</td></tr>', $value->option_name, size_format( $value->option_value_length, 2 ) );
+		$url            = esc_url_raw(
+			add_query_arg(
+				array(
+					'action'       => 'perflab_aao_update_autoload',
+					'_wpnonce'     => $nonce,
+					'option_name'  => $value->option_name,
+					'autoload'     => 'false',
+					'value_length' => esc_attr( $value->option_value_length ),
```

Comment:
Maybe call this `option_size` instead? Similar to how the table is labelled.

It's a clever solution to store the size so that we can still show it. However, this value may get out of date, so I wonder whether we should instead implement this more dynamically. Last but not least it feels a bit strange that the option size has to be passed to change the autoload value.